src/SessionSetup/RetrieveInstancesWidget.cpp

```diff
@@ -31,21 +32,37 @@ namespace orbit_session_setup {
 using orbit_ggp::Client;
 using orbit_ggp::Instance;
 using orbit_ggp::Project;
+using LoadProjectsAndInstancesResult = RetrieveInstances::LoadProjectsAndInstancesResult;
+
+namespace {
+
+std::optional<Project> GetQSettingsProject() {
```

Comment:
Probably this should nevertheless be `[[nodiscard]]`, shouldn't it?

Reply:
yes!